package.json

```diff
@@ -6,23 +6,26 @@
   "scripts": {
     "frontend": "nx serve frontend",
     "deploy": "nx deploy frontend",
-    "buildall": "nx run-many --target=build"
+    "buildall": "nx run-many --target=build",
+    "mini-ci": "nx affected -t test,lint --max-warnings=0"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.18.6",
     "@emotion/babel-plugin": "11.11.0",
     "@nx/cypress": "16.4.3",
     "@nx/eslint-plugin": "16.4.3",
-    "@nx/jest": "16.4.3",
-    "@nx/js": "16.4.3",
+    "@nx/jest": "16.5.1",
+    "@nx/js": "16.5.1",
     "@nx/linter": "16.4.3",
+    "@nx/plugin": "^16.5.1",
     "@nx/react": "16.4.3",
     "@nx/vite": "16.4.3",
     "@nx/web": "16.4.3",
     "@nx/webpack": "16.4.3",
     "@nx/workspace": "16.4.3",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
     "@svgr/webpack": "8.0.1",
+    "@swc-node/register": "~1.4.2",
```

Comment:
What is this used for?

Reply:
Nx seems to be using this to run executor (sync-repo)

Reply:
We can remove it and switch to the "slower" tsc